service/src/main/kotlin/fi/espoo/evaka/invoicing/domain/FeeDecisions.kt

```diff
@@ -187,7 +187,10 @@ data class FeeDecisionDetailed(
 
     val requiresManualSending
         get(): Boolean {
-            if (decisionType !== FeeDecisionType.NORMAL || headOfFamily.forceManualFeeDecisions) {
+            // Restricted will be sent to allow fast receiving via suomi.fi e-channel.
+            if (headOfFamily.restrictedDetailsEnabled) {
+                return false
+            } else if (decisionType !== FeeDecisionType.NORMAL || headOfFamily.forceManualFeeDecisions) {
```

Comment:
It's not this part that needs to be changed. Decisions where `decisionType !== FeeDecisionType.NORMAL || headOfFamily.forceManualFeeDecisions` is `true` should still be manually sent. It's the next condition that needs to take into account that people with `restrictedDetailsEnabled` have no address data.

Reply:
Good catch!